pkgs/applications/networking/whoogle-search/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "whoogle-search";
+  version = "0.7.1";
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    hash = "sha256-jn5Plm0yps3yww9Uj3Z77HejuAEMhw1/MqJpCcn7Vv8=";
+  };
+
+  patchPhase = ''
+    sed -i 's/==/>=/' requirements.txt
+  '';
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pysocks
+    python-dotenv
+    attrs
+    requests
+    beautifulsoup4
+    cachelib
+    certifi
+    cffi
+    chardet
+    click
+    cryptography
+    defusedxml
+    flask
+    flask-session
+    idna
+    itsdangerous
+    jinja2
+    markupsafe
+    more-itertools
+    packaging
+    pluggy
+    py
+    pycodestyle
+    pycparser
+    pyopenssl
+    pyparsing
+    python-dateutil
+    soupsieve
+    stem
+    urllib3
+    waitress
+    wcwidth
+    werkzeug
+    pytest
+  ];
+
+  # Do not start tests
```

Comment:
Why?

Reply:
Unfortunately the tests require the outside internet connection. I've added a comment.